Godeps/_workspace/src/github.com/steakknife/hamming/README.md

```diff
@@ -0,0 +1,3 @@
+Copyright (c) 2014 Barry Allard
+
+MIT license
```

Comment:
This is not bikeshedding. I've checked this with lawyers multiple times. As things stand today, it is still required to have the full text of licenses along with the relevant code base. 




I already did the work for you. If you can't press one button, fine. We'll add it ourselves here. You're foisting much more work unto us -- every time we vendor -- but I don't care about arguing you on this



—
Sent from Mailbox

On Fri, Apr 17, 2015 at 10:11 AM, Barry Allard <notifications@github.com>
wrote:

>> @@ -0,0 +1,3 @@
>> +Copyright (c) 2014 Barry Allard
>> +
>> +MIT license
> @jbenet Tyrannical, bureaucratic "standards" people exist to foist unnecessary bikeshedding FUD and work on others while contributing no real value.  I refuse to set this precedent unless you have a really good reason.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ipfs/go-ipfs/pull/1085/files#r28611967